biz.aQute.bndlib/src/aQute/bnd/classfile/ConstantPool.java

```diff
@@ -888,4 +986,14 @@ public void write(DataOutput out) throws IOException {
 			}
 		}
 	}
+
+	@Override
+	public Iterator<Object> iterator() {
+		return Arrays.asList(pool)
+			.iterator();
+	}
+
+	public Stream<Object> stream() {
+		return Stream.of(pool);
```

Comment:
`return Arrays.stream(pool, 1, pool.length);` to avoid returning the invisible 0-index entry.